### ISSUE TITLE
Highmaps: Zoom not working properly

### ISSUE THREAD

**Issue by sagargurtu** [highcharts/highcharts#6056]
#### Expected behaviour
Map should not zoom in on click and drag when zoomType is null.

#### Actual behaviour
Click and drag at any point on the map. The map zooms in automatically. Zoom out on mouse scroll works only after a certain amount of scrolling.

#### Live demo with steps to reproduce
http://jsfiddle.net/txr7oy2h/1/

#### Affected browser(s)
Chrome

**Comment by TorsteinHonsi** [comment]
With Highmaps, the [mapNavigation](http://api.highcharts.com/highmaps/mapNavigation) settings override zoomType: http://jsfiddle.net/highcharts/txr7oy2h/2/

**Comment by TorsteinHonsi** [comment]
PS: `chart.zoomType` is not part of the Highmaps API at all: http://api.highcharts.com/highmaps/chart
